mathesar/models/query.py

```diff
@@ -120,6 +120,12 @@ class UIQuery(BaseModel, Relation):
         unique=True,
     )
 
+    description = models.CharField(
+        max_length=280,
```

Comment:
Why have a max length here? Why isn't it just a `TextField`?

Reply:
I thought of making it a `TextField` but then I thought users won't have descriptions that long for describing an exploration.

Reply:
t think we should try to be consistent with table / schema descriptions, which are Postgres comments and can be arbitrarily long.

Reply:
Makes sense I'll change it to `TextField`.